central_service/app/frontend/app/templates/faqs/lambda-application/check-running.hbs

```diff
@@ -0,0 +1,177 @@
+      <div class="box-body">
+        <div class="row">
+          <div class="col-md-12">
+            <h1>Check Running Applications</h1>
+
+            <p>
+              To view your Lambda Applications you can follow the steps below.
+            </p>
+
+            <div class="row margin-bottom">
+              <div class="col-sm-6">
+                <div class="row margin-bottom">
+                  <div class="col-sm-3">
+                    <a href="#" data-toggle="modal" data-target="#LoginPage">
+                      <img class="img-responsive" src="assets/img/faqs/login-sm.png" alt="Login" title="Login" >
+                    </a>
+                    <p class="text-center">Step 1</p>
+                  </div>
+                  <div class="col-sm-1 text-center">
+                    <span class="fa fa-arrow-right"></span>
+                  </div>
+                  <div class="col-sm-3">
+                    <a href="#" data-toggle="modal" data-target="#createStep2">
+                      <img class="img-responsive" src="assets/img/faqs/lambda-application/create-sm.png" alt=" Page" title="Home Page">
+                    </a>
+                    <p class="text-center">Step 2</p>
+                  </div>
+                  <div class="col-sm-1 text-center">
+                    <span class="fa fa-arrow-right"></span>
+                  </div>
+                  <div class="col-sm-3">
+                    <a href="#" data-toggle="modal" data-target="#applist">
+                      <img class="img-responsive" src="assets/img/faqs/lambda-application/check-running/step3-sm.png" alt="List Applications" title="List Applications">
+                    </a>
+                    <p class="text-center">Step 3</p>
+                  </div>
+                </div>
+                <div class="row ">
+                  <div class="col-sm-12 col-md-offset-9">
+                    <span class="fa fa-arrow-down"></span>
+                  </div>
+                </div>
+                <div class="row ">
+                  <div class="col-sm-3 col-md-offset-8">
+                    <a href="#" data-toggle="modal" data-target="#apprunning">
+                      <img class="img-responsive" src="assets/img/faqs/lambda-application/details-sm.png" alt="Application Running"  title="Application Running" >
+                    </a>
+                    <p class="text-center">Step 4</p>
+                  </div>
+                </div>
+              </div>
+            </div><!--row-->
+
+            <ol>
+              <li>
+                Log into the LoD Service VM using your <a href="https://accounts.okeanos.grnet.gr/ui/api_access">~okeanos token</a>.
+              </li>
+
+              <li>
+                From the sidebar, select "Applications".
+              </li>
+
+              <li>
+                From the list of Lambda Applications that appear, click "Details" to see more information about a specific Application.
```

Comment:
The 4th step is missing from this list. Also, for the 3rd step, under the image description 'On the list' is used, while here is written 'From the list'.